tests/tests_type_checking/test_lightning_module.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.demos.boring_classes import BoringModel
+
+
+def test_load_from_checkpoint_type(tmp_path: Path) -> None:
```

Comment:
I don't think a test is required.

Reply:
I agree. Types are checked by running mypy in the source code

Reply:
removed test